package.json

```diff
@@ -42,26 +42,26 @@
     "execa": "^5.1.1",
     "fs-extra": "^10.0.0",
     "jest": "^27.0.6",
-    "lint-staged": "^11.1.1",
+    "lint-staged": "^11.1.2",
     "minimist": "^1.2.5",
     "node-fetch": "^2.6.1",
     "npm-run-all": "^4.1.5",
-    "playwright-chromium": "^1.13.1",
+    "playwright-chromium": "^1.14.0",
     "prettier": "2.3.2",
     "prompts": "^2.4.1",
     "rimraf": "^3.0.2",
     "semver": "^7.3.5",
     "sirv": "^1.0.14",
-    "ts-jest": "^27.0.4",
-    "ts-node": "^10.1.0",
```

Comment:
It's `ts-node`!

Reply:
I set it to minor for now...
we could also add it to ignore in renovate config :thinking: